jina/executors/crafters/__init__.py

```diff
@@ -8,11 +8,10 @@
 
 
 class BaseCrafter(BaseExecutor):
-    """A :class:`BaseCrafter` craft the content of `Document` or `Chunk`. It can be used for preprocessing,
-    segmenting etc.
+    """A :class:`BaseCrafter` encodes the content of `Document` or `Chunk`. It can be used for preprocessing,
```

Comment:
`encodes` is not the right word, for that we use `encoders`. May we use `transforms`?

Reply:
Updated to `transforms`